docs/lifecycle/application/application.md

```diff
@@ -11,13 +11,13 @@ However, here are the most common and recommended things you need to keep track
 - Configure a router: to easily manage your application flow and transitions between sections.
 - Include a unit testing tool: to ensure the quaility of your application and avoid side effects.
 
-Below, you will find out a set of Vue tools that will leverage your setup process and give you a better idea on how to setup your application.
+Below, you will find out a set of Vue.js tools that will leverage your setup process and give you a better idea on how to setup your appliaction.
```

Comment:
We have a typo here when solving the Vue.js naming convention

Reply:
What'd be the naming convention?
I saw vuejs, Vue, Vue JS, vue and tried to unified into Vue.js

Reply:
I think @josepenagosGL refers to _appliaction_

Reply:
@juangiraldoGL is right

Reply:
done